lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -25,7 +25,9 @@ module.exports = function(
   DrivesModel,
   SelectionStateModel,
   WarningModalService,
-  DriveConstraintsModel) {
+  DriveConstraintsModel,
+  AnalyticsService
```

Comment:
You have to add this to the module declaration as well, at `lib/gui/components/drive-selector/drive-selector.js`